datasets/md/terror_sanctions/crawler.py

```diff
@@ -6,15 +6,28 @@
 from zavod import Context
 from zavod import helpers as h
 
+SPLITS = [
+    "si",
+    ";",
+    "sau",
+    "a)",
+    "b)",
+    "c)",
+    "d)",
+    "Aproximativ ",
+    "Intre",
+    "între",
+    "și",
+    "la",
+    "din pasaport fals",
+    "presupusă:",
+]
 
-FORMATS = ["%d.%m.%Y"]
-SPLITS = ["si", ";", "sau", "a)", "b)", "c)"]
 
-
-def parse_birth_dates(string: str) -> List[str]:
+def parse_birth_dates(context: Context, string: str) -> List[str]:
```

Comment:
Similar as with -12, I wonder if this should return unparsed dates and feed them into `h.apply_date`.